assets/js/consent-mode/consent-mode.test.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Consent Mode tests
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
```

Comment:
```suggestion
 * Site Kit by Google, Copyright 2024 Google LLC
```
Still a bit early xD